src/web/src/app.tsx

```diff
@@ -0,0 +1,362 @@
+  const paramsList = props.exp.parameters().map((p) => <li key={p}>{p}</li>);
+  const paramsDiv =
+    paramsList.length > 0 ? (
+      <ul className="study-meta">
+        <span>Parameters: </span>
+        {paramsList}
+      </ul>
+    ) : (
+      <></>
+    );
+  const classes =
+    'list-group-item ' +
+    (props.exp.isMajorGroup() ? 'major-exp-item' : 'exp-item');
+  return (
+    <li className={classes}>
+      {props.exp.name()} ({props.exp.weight()}%)
+      <FeatureList
+        title="Enabled features"
+        className="enabled-feature"
+        features={props.exp.enabledFeatures()}
+      />
+      <FeatureList
+        title="Disabled features"
+        className="disabled-feature"
+        features={props.exp.disabledFeatures()}
+      />
+      {paramsDiv}
+    </li>
+  );
+}
+
+export function PropertyList(props: {
+  caption: string;
+  list: string[] | null | undefined;
+}): JSX.Element {
+  if (props.list == null || props.list.length === 0) return <></>;
+  return (
+    <ul className="study-meta">
+      <span>{props.caption}: </span>
+      {props.list.map((c) => (
+        <li key={c}>{c}</li>
+      ))}
+    </ul>
+  );
+}
+
+export function IncludeExcludeList(props: {
+  caption: string;
+  include: string[] | null | undefined;
+  exclude: string[] | null | undefined;
+}): JSX.Element {
+  return (
+    <>
+      <PropertyList caption={props.caption} list={props.include} />
+      <PropertyList caption={'Exclude' + props.caption} list={props.exclude} />
+    </>
+  );
+}
+
+export function StudyItem(props: {
+  study: StudyModel;
+  filter: StudyFilter;
+}): JSX.Element {
+  const filter = props.study.filter();
+  return (
+    <div className="card mb-3">
+      <div className="card-header">
+        <a target="_blank" href={props.study.getConfigUrl()} rel="noreferrer">
+          {props.study.name()}
+        </a>
+      </div>
+      <div className="card-body">
+        <ul className="list-group list-group-flush">
+          {props.study.experiments(props.filter).map((e) => (
+            <ExperimentItem key={e.name()} exp={e} />
+          ))}
+        </ul>
+      </div>
+      <div className="card-footer">
+        <PropertyList caption="Channels" list={props.study.channels()} />
+        <PropertyList caption="Platforms" list={props.study.platforms()} />
+        <IncludeExcludeList
+          caption="Countries"
+          include={filter?.country}
+          exclude={filter?.exclude_country}
+        />
+        <IncludeExcludeList
+          caption="Hardware class"
+          include={filter?.hardware_class}
+          exclude={filter?.exclude_hardware_class}
+        />
+      </div>
+    </div>
+  );
+}
+
+function getCurrentSeedType(searchParams: URLSearchParams): SeedType {
+  return (
+    stringToSeedType(searchParams.get('seed') ?? 'PRODUCTION') ??
+    SeedType.PRODUCTION
+  );
+}
+
+export function NavItem(props: { type: SeedType }): JSX.Element {
```

Comment:
nit: you don't need to put the function return type, it's inferred and obvious for React functions.

Reply:
heh, it's because of eslint settings. I'm going to relax them

Reply:
Removed.